arangod/Cluster/ClusterInfo.cpp

```diff
@@ -3134,6 +3138,13 @@ Result ClusterInfo::createCollectionsCoordinator(
             return r;
           }
         }
+      } else {
+        LOG_TOPIC("98675", WARN, Logger::CLUSTER)
```

Comment:
Is this something you want someone to be called at night?

Reply:
It doesn't. ERROR creates a ticket, FATAL wakes up
https://github.com/arangodb/arangodb/blob/devel/ERROR_LEVELS.md

Reply:
Ok 👌🏽